src/replay/consensus.zig

```diff
@@ -32,6 +32,7 @@ const SlotTracker = sig.replay.trackers.SlotTracker;
 const EpochTracker = sig.replay.trackers.EpochTracker;
 
 pub const isSlotDuplicateConfirmed = sig.consensus.tower.isSlotDuplicateConfirmed;
+const VoteThreshold = sig.consensus.progress_map.ForkStats.VoteThreshold;
```

Comment:
Kind of odd to throw this import in between two exports.

Reply:
[78704ae](https://github.com/Syndica/sig/pull/851/commits/78704ae8dc992eec5ba9c09b844d04d6591026c7)